src/main/java/org/apache/datasketches/tuple/arrayofdoubles/HeapArrayOfDoublesQuickSelectSketch.java

```diff
@@ -63,7 +63,8 @@
   HeapArrayOfDoublesQuickSelectSketch(final int nomEntries, final int lgResizeFactor,
       final float samplingProbability, final int numValues, final long seed) {
     super(numValues, seed);
-    nomEntries_ = nomEntries;
+    lgNomEntries_ =
+        org.apache.datasketches.Util.simpleLog2OfLong(org.apache.datasketches.Util.ceilingPowerOf2(nomEntries));
```

Comment:
perhaps this fully qualified name is avoidable?

Reply:
I will fix using import static ...method.